### PR TITLE
Style paper key entry during provisioning

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers 

device-2016-04-19-171934

Also adds some flow state entries and fixes a few bugs.

### CODE REVIEW

File: shared/common-adapters/input.native.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ class Input extends Component<void, Props, State> {
           onChange={this.props.onChange}
           onChangeText={text => { this.setState({text}); this.props.onChangeText && this.props.onChangeText(text) }} />
         {IOS && <HorizontalLine focused={this.state.inputFocused}/>}
-        {this.props.errorText && <Text type='Error' style={{...errorText, ...this.props.errorStyle}}>{this.props.errorText}</Text>}
+        {!!(this.props.errorText) && <Text type='Error' style={{...errorText, ...this.props.errorStyle}}>{this.props.errorText}</Text>}
```
Comment (cjb) [inline shared/common-adapters/input.native.js]: via @MarcoPolo, this fixes `RawText "" must be wrapped in an explicit <Text> component.` when this.props.errorText is blank.

File: shared/login/register/paper-key/index.js [hunk 2]
```diff
@@ -34,5 +34,5 @@ PaperKey.propTypes = {
 }
 
 export default connect(
-  (state, ownProps) => ownProps.mapStateToProps(state)
+  state => ({waitingForResponse: state.login.waitingForResponse})
```
Comment (chrisnojima) [inline shared/login/register/paper-key/index.js]: i'm confused how this works. this isn't passed down to the render, so do we need it? same below

Comment (cjb) [inline shared/login/register/paper-key/index.js]: You're right, doesn't work, though we do need to use waitingForResponse to show button spinners -- I'll ask you tomorrow for the right way to get it in a prop.

Comment (chrisnojima) [inline shared/login/register/paper-key/index.js]: injecting it this way is fine, it just needs to be passed into the render component

**Comment by cjb** [comment]
@chrisnojima Am passing waitingForResponse through properly now, tested working. Thanks!

**Comment by chrisnojima** [comment]
👍

**Pull request merged**